contracts/scripts/deploy.ts

```diff
@@ -15,8 +16,12 @@ async function main() {
   const fundingRoundFactory = await FundingRoundFactory.deploy(
     maciFactory.address
   )
-  await fundingRoundFactory.deployed()
-  await maciFactory.transferOwnership(fundingRoundFactory.address)
+  await fundingRoundFactory.deployTransaction.wait()
+
+  const transferReceipt = await maciFactory.transferOwnership(
+    fundingRoundFactory.address
+  )
+  await transferReceipt.wait()
```

Comment:
TBH I'm not sure if this one is necessary. Not sure if we just need to `wait()` on deploy txs or if we need to `wait()` on every tx.